### DISCUSSION TITLE
Application not launching when converting from 32bit to 64bit in VS 2019

### DISCUSSION THREAD

**Discussion by ThejaswiniN** [community/community#76207] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Recently I have converted the application from 32bit to 64bit in VS 2019,all sources are saying the application is of 64bit but I'm unable to launch the application ,
![64bit Cmd](https://github.com/community/community/assets/74773490/16d02ebc-e8f3-44df-b958-1f8ff98fd91c)
and I am not able to debug the application because the application need to be attach to process to debug, when searched in the event viewer, I am encountering BEX64 error as attached 
<img width="691" alt="BEX64 error" src="https://github.com/community/community/assets/74773490/3154e966-9b56-4b56-9c25-889a66eff21b">
, and have made sure that all the dependent DLL's are of 64bit  , Can anyone please help me to resolve this issue.

I am able to open the application from visual Studio 2019, but not able to launch from exe
<img width="923" alt="application launching" src="https://github.com/community/community/assets/74773490/0cabba64-9735-4f04-af7f-392ff10e1514">

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @ThejaswiniN! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
